toast-tk-runtime/src/main/java/io/toast/tk/runtime/result/ObjectResultHandler.java

```diff
@@ -9,10 +14,36 @@
     public ObjectResultHandler(IActionItemRepository objectRepository) {
         super(objectRepository);
     }
-    
+
+	@SuppressWarnings({ "unchecked", "rawtypes" })
+    private String resultBuetifier(Object value) {
+    	String res = "";
+		
+		try {
+			res = (String) ((List)value).stream()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unnecessary cast to "String". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1905)